libgstd/gstd_parser.c

```diff
@@ -270,11 +270,17 @@ gstd_parser_create (GstdSession * session, GstdObject * obj, gchar * args,
   if (ret)
     goto out;
 
-  gstd_object_read (obj, name, &new);
-
-  if (NULL != new) {
-    gstd_object_to_string (new, response);
-    g_object_unref (new);
+  /* Events doesn't create gstd objects */
+  if (0 != strcmp (name, "eos")
+      && 0 != strcmp (name, "seek")
+      && 0 != strcmp (name, "flush_start")
+      && 0 != strcmp (name, "flush_stop")) {
```

Comment:
I was thinking on something more generic to avoid listing each of the events like

 ```suggestion
  if ( !GSTD_IS_EVENT_CREATOR(obj)){
```

or event more generic to any type of object that doesn't have a reader

```suggestion
  if (!GSTD_IS_NO_READER(obj->reader)) {
```